GOVERNANCE.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Knative Governance Framework"
+linkTitle: "Governance"
+weight: 20
+type: "docs"
+aliases:
+- /contributing/governance/
+---
+
+Knative Governance is broken into three components. At a high level, the Knative
+Steering Committee (KSC) is the "root" level governance body, but the Knative
+Trademark Committee (KTC) operates within a protected governance umbrella in
+parallel with the KSC. As such, the KSC may enforce Code of Conduct sanctions or
+the like against members of the KTC, but may not alter the charter or governance
+of the KTC rules. The Tech Oversight Committee (TOC) operates entirely within
+the governance definitions delineated by the KSC.
+
+## Community and Operations
+
+The [Knative Steering Committee](./STEERING-COMMITTEE.md) is responsible for the
+general health of the Knative community. Responsibilities include maintaining
+the governance framework, code of conduct, community meetings, communication
+platforms, and test resources.
+
+
+## Trademarks
+
+The [Knative Trademark Committee](./TRADEMARK-COMMITTEE.md) is responsible for
+management, licensing, and conformance verification of the Knative trademark,
```

Comment:
s/the Knative trademark/of Knative logos, names, and trademarks,/

Reply:
Done.